### PR TITLE
[RLlib] DDPG PyTorch actor-model was missing sigmoid layer

### PR DESCRIPTION
**Pull request by sven1977**
Our DDPG PyTorch action model was missing the mandatory sigmoidal layer (to squash actions into the action bounds). This PR fixes this problem.
This was also the reason, DDPG torch would not be able to learn MuJoCo env (e.g. HalfCheetah).
Learning capabilities on HalfCheetah have been confirmed.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: rllib/agents/ddpg/ddpg_torch_model.py [hunk 1]
```diff
@@ -79,6 +79,22 @@ def __init__(self,
                 initializer=torch.nn.init.xavier_uniform_,
                 activation_fn=None))
 
+        # Use sigmoid to scale to [0,1], but also double magnitude of input to
```
Comment (sven1977) [inline rllib/agents/ddpg/ddpg_torch_model.py:82]: This was missing entirely.

File: rllib/agents/ddpg/ddpg_torch_policy.py [hunk 2]
```diff
@@ -163,9 +163,12 @@ def ddpg_actor_critic_loss(policy, model, _, train_batch):
 def make_ddpg_optimizers(policy, config):
     # Create separate optimizers for actor & critic losses.
     policy._actor_optimizer = torch.optim.Adam(
-        params=policy.model.policy_variables(), lr=config["actor_lr"])
+        params=policy.model.policy_variables(),
```
Comment (sven1977) [inline rllib/agents/ddpg/ddpg_torch_policy.py:166]: Make Adam optimizer exactly the same in torch AND tf (torch default eps is 1e-8!).

Comment (sven1977) [reply rllib/agents/ddpg/ddpg_torch_policy.py:166]: Compare with SAC (where we do this, too).

File: rllib/agents/ddpg/tests/test_ddpg.py [hunk 3]
```diff
@@ -1,11 +1,19 @@
 import numpy as np
+import re
```
Comment (sven1977) [inline rllib/agents/ddpg/tests/test_ddpg.py:2]: Add loss function test (1:1 comparison with tf version) for DDPG.

**Review by ericl — APPROVED** [review]
